downloads/swan-lake-release-notes/swan-lake-2201.7.3/RELEASE_NOTE.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.7.3 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201-7-3/
+active: 2201-7-3
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-7-3
+    - /downloads/swan-lake-release-notes/2201.7.3/
+    - /downloads/swan-lake-release-notes/2201-7-3-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-7-3-swan-lake
+---
+
+## Overview of Ballerina Swan Lake 2201.7.3
+
+<em>Swan Lake 2201.7.3 is the third patch release of Ballerina 2201.7.0 (Swan Lake Update 7) and it includes a new set of bug fixes to the language, standard library, and tooling.</em>
```

Comment:
```suggestion
<em>Swan Lake 2201.7.3 is the third patch release of Ballerina Swan Lake Update 7 (2201.7.0) and it includes a new set of bug fixes to the language, Ballerina library, and tooling.</em>
```
The new format for writing the release version is `Swan Lake Update 8 (2201.8.0)`. Do we have tooling fixes also in this release?